### ISSUE TITLE
After updating to pw 2.6 stable, adding new users not possible others than superuser

### ISSUE THREAD

**Issue by apeisa** [apeisa/UserGroups#42]
Trying to add new user (as a non superuser) gives me error:

You do not have permission to execute this module - ProcessPageAdd

I tracked it down to this line: https://github.com/apeisa/UserGroups/blob/master/UserGroupsHooks.module#L157

If we comment that line, we don't get any errors. That cannot be real solution, since we are using before hook, so changing return value without replace is useless (as far as I know).

This error must be due the changes in how ProcessWire handles module permissions. Since we don't actually load ProcessPageAdd anywhere (or do we?), this is much harder to track. 

Any ideas @teppokoivula ? Have you tried UserGroups in 2.6 yet? All the other parts seem to work nicely, just this "non superuser adding new users" is causing troubles.

**Comment by teppokoivula** [comment]
@apeisa: nope, haven't used this with 2.6 yet. It's kind of a long shot, but I'm wondering if the "multi-template/parent user support" broken something here?

I'm kind of fully booked right now, so it might take a while before I can take a closer look. Let me know if there's any news though :)

**Comment by apeisa** [comment]
Hmm, this got even weirder. The user that I used to add new users didn't have any user groups set, only role "site-admin" that gave him bunch of permissions (user-admin, page-edit, profile-edit etc). Right after I gave him one user group, it started working... Without user groups hook installed it works right away.

**Comment by apeisa** [comment]
"multi template/parent" was my first suspect also, but this is not it (although that might require some love from us at some point).

**Comment by teppokoivula** [comment]
Any news about this? I'm just about to update one 2.5 site using UserGroups to 2.6, will take a closer look if the issue still persists :)

**Comment by apeisa** [comment]
Nope, run out of time debugging and since it does work when having user group assigned, didn't bother to dwell deeper. Let me know how it goes for you!
